src/com/github/johnthagen/cppcheck/CppCheckInspectionImpl.java

```diff
@@ -45,72 +55,121 @@ private static ProblemHighlightType severityToHighlightType(@NotNull final Strin
         }
     }
 
+    // TODO: make configurable
+    private static final boolean VERBOSE_LOG = false;
```

Comment:
I will probably adjust this to just give information what the execution itself and make the XML data available via an IDE action. That way it's still possible to track the invocation and also get the non-truncated output (the event log has maximum size for entries).